shell/app/node_main.cc

```diff
@@ -107,8 +108,12 @@ int NodeMain(int argc, char* argv[]) {
 
   auto os_env = base::Environment::Create();
   bool node_options_enabled = electron::fuses::IsNodeOptionsEnabled();
+  if (!node_options_enabled) {
+    os_env->UnSetVar("NODE_OPTIONS");
```

Comment:
Node already has `kDisableNodeOptionsEnv`, which we enable. It seems like a Node bug that there are workarounds for this? Can we fix this in Node?

Reply:
`kDisableNodeOptionsEnv` means disabling the `NODE_OPTIONS` env, it is probably arguable whether it should disable other envs too, but anyway I'll bring this up to Node and will patch upstream if they agree so.

In the long term I think what we really need is a way to guarantee there is no way to inject code into an app, but I don't have a good idea how to implement it. I'll discuss with Node team too.